randovania/games/dread/json_data/header.txt

```diff
@@ -111,31 +111,222 @@ Templates
               Bomb and Water Bomb Jump (Beginner)
               Cross Bomb and Water Bomb Jump (Intermediate)
 
+* Open Charge Door:
+      Any of the following:
+          Shoot Charge Beam
+          Knowledge (Beginner) and Lay Power Bomb
+
+* Fight Silver Robot:
+      Any of the following:
+          Knowledge (Beginner) and Lay Power Bomb
```

Comment:
I believe that Knowledge checks such as this, in the context of fights, are better off as Combat tricks. @henriquegemignani can verify but iirc it's used that way in the other games

Reply:
I checked with exo, and flamethrower instead of power beam is listed as both knowledge and combat

Reply:
in that case, use both :+1: